wallet/src/key_chain/leaf_key_chain/mod.rs

```diff
@@ -418,6 +419,16 @@ impl LeafKeyChain {
         false
     }
 
+    pub fn is_destination_mine(&self, dest: &Destination) -> bool {
+        match dest {
+            Destination::Address(pkh) => self.is_public_key_hash_mine(pkh),
+            Destination::PublicKey(pk) => self.is_public_key_mine(pk),
+            Destination::AnyoneCanSpend
+            | Destination::ScriptHash(_)
+            | Destination::ClassicMultisig(_) => false,
```

Comment:
This can also have another public key hash function

Reply:
I'm not sure what steps we should take to support this... but whether now or later, this should have some TODO on it at least.

Reply:
TODO added.